### PR TITLE
Add OSX fix for contacts sync

### PR DESCRIPTION
**Pull request by skjnldsv**
Fix nextcloud/contacts#399

### CODE REVIEW

**Comment by skjnldsv** [comment]
Thanks a lot @MariusBluem :)
There is a reference forum thread,  shall we link it from the docs? https://help.nextcloud.com/t/discussion-adressbooks-sync-with-macos/25760/14

**Comment by BernieO** [comment]
This doesn't work for me - and I doubt that this is a solution for Mac OS X not being able to sync more then one addressbook.
For details see my comment here:
https://github.com/nextcloud/contacts/issues/399#issuecomment-468992259

File: admin_manual/installation/nginx.rst [hunk 1]
```diff
@@ -374,3 +374,14 @@ block shown above not located **below** the:
...
+Contacts only syncing the first addressbook with macOS
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+1. Comment out any of those lines if you have them set on your nginx config:
+.. code-block:: nginx
+
+        error_page 403 /core/templates/403.php;
+        error_page 404 /core/templates/404.php;
+        
+2. Change ``fastcgi_intercept_errors`` from ``on`` to ``off``.
```
Comment (MorrisJobke) [inline admin_manual/installation/nginx.rst]: Those files don't exist anymore since Nextcloud 12. 👎

**Comment by skjnldsv** [comment]
Closing as it apparently doesn't fix anything :(

**Pull request closed without merging**